src/base/http/connection.cpp

```diff
@@ -106,6 +106,19 @@ void Connection::read()
             }
             return;
 
+        case RequestParser::ParseStatus::BadMethod:
+            {
+                LogMsg(tr("Bad Http request method, closing socket. IP: %1. Method: \"%2\"")
```

Comment:
The similar problem was discussed recently. I feel it is a bad idea to add log message in such a case which could allow allow an attacker to easily overload the qBittorrent server by producing log files, simply by sending obviously invalid requests.

Reply:
Added another commit for this.